runtime/world.js

```diff
@@ -288,13 +288,6 @@ function World() {
     }
   });
 
-  /**
-   * this initiates the driver before every scenario is run
-   */
-  Before(function () {
-      global.driver = getDriverInstance();
-  });
-
```

Comment:
@larryg01 if it helps, we are already running tests with the modified version.

Reply:
yes it will work on but i am asking if you ran the example search test without it
if you did and the test ran without falling over, cause remember its running the function 3 times then great but it was put in to ensure that repeated test like searches work.